src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp

```diff
@@ -28,16 +29,16 @@ void OpSummaryDestroyer::initialize(OpSummary *p) {
 OpSummary::OpSummary() {
     reportFilename = CommonTestUtils::OP_REPORT_FILENAME;
     // TODO: replace to get_available_opsets()
-    opsets.push_back(ngraph::get_opset1());
-    opsets.push_back(ngraph::get_opset2());
-    opsets.push_back(ngraph::get_opset3());
-    opsets.push_back(ngraph::get_opset4());
-    opsets.push_back(ngraph::get_opset5());
-    opsets.push_back(ngraph::get_opset6());
-    opsets.push_back(ngraph::get_opset7());
-    opsets.push_back(ngraph::get_opset8());
-    opsets.push_back(ngraph::get_opset9());
-    opsets.push_back(ngraph::get_opset10());
+    opsets.push_back(ov::get_opset1());
```

Comment:
Can you use `ov::get_available_opsets()` method?

Reply:
Fixed :) I used this API in other places, here i decided to take only version